src/Symfony/Component/Security/Core/Authorization/Voter/Vote.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+class Vote
+{
+    use AccessTrait;
+
+    private $reason;
+    private $parameters;
+
+    private function __construct(int $access, string $reason = '', array $parameters = [])
+    {
+        $this->access = $access;
+        $this->reason = $reason;
+        $this->parameters = $parameters;
+    }
+
+    public static function create(int $access, string $reason = '', array $parameters = []): self
```

Comment:
```suggestion
    /**
     * @var int $access One of the constant of VoterInterface::ACCESS_
     */
    public static function create(int $access, string $reason = '', array $parameters = []): self
```

Reply:
I added the phpdoc where you suggested with a wording respecting the rest of the codebase.